lib/src/rpc/dial.dart

```diff
@@ -234,11 +234,11 @@ Future _logConnectionStats(Stopwatch webrtcDialSW, RTCPeerConnection peerConnect
     if (stat.type == 'candidate-pair' && stat.values['nominated']) {
       // Use 'lastPacketSentTimestamp' on candidate pair to estimate when the
       // pair was nominated.
-      final double lpst = stat.values['lastPacketSentTimestamp'];
+      final double lpst = stat.values['lastPacketSentTimestamp'] ?? 0;
```

Comment:
[nit] not set on the `?? 0` if there is a better default value - or if we should set it to null to do some exception handling than that may be preferred. I'll leave that up to you though!

Reply:
Ah I think I'm fine with the `?? 0`.